iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SignUpViewModel.swift
+//  
+//
+//  Created by Raynelle Francisca on 27/11/2020.
+//
+
+import Foundation
+import SwiftUI
+import shared
+import Components
+
+public final class SignUpViewModel: ObservableObject {
+
+	var email: String
```

Comment:
Does this need to be variable? I can't see it being reassigned anywhere. Does it also need to be internal or can it be private?

Reply:
I think we need that since that’s the email that’s being passed from the previous view `SignUpWithEmail` to the next view `SignUpWithPassword` and is injected via the viewModel